modules/mod_ginger_base/templates/head/favicon.tpl

```diff
@@ -1,5 +1,5 @@
-<link rel="apple-touch-icon-precomposed" sizes="144x144" href="/lib/images/apple-touch-icon-144x144.png" />
-<link rel="apple-touch-icon-precomposed" sizes="152x152" href="/lib/images/apple-touch-icon-152x152.png" />
+<link rel="apple-touch-icon" sizes="144x144" href="/lib/images/apple-touch-icon-144x144.png" />
+<link rel="apple-touch-icon" sizes="152x152" href="/lib/images/apple-touch-icon-152x152.png" />
```

Comment:
I've tested it on my iphone, and there is no visual change. The precomposed was for the purpose of ignoring the styling of ios (like shiny stuff and shadows) but i think this is no longer added by apple. See screenshot for test.

![img_2797](https://cloud.githubusercontent.com/assets/5102362/24759301/60e78a26-1ae5-11e7-9287-44fdd02249d7.PNG)